lib/wallet/src/Cardano/Wallet/Write/Tx.hs

```diff
@@ -826,10 +896,22 @@ evaluateMinimumFee era pp tx kwc =
         Shelley.evaluateTransactionFee pp tx nKeyWits
 
     bootWitnessFee :: Coin
-    bootWitnessFee =
-        if nBootstrapWits > 0
-        then error "evaluateMinimumFee: bootstrap witnesses not yet supported"
-        else mempty
+    bootWitnessFee = Coin $
+        intCast $ feePerByte * bytes
+      where
+        bytes :: Natural
+        bytes = sizeOf_BootstrapWitnesses $ intCast nBootstrapWits
+
+        -- Matching implementation in "Cardano.Wallet.Shelley.Transaction".
+        -- Equivalence is tested in property.
+        sizeOf_BootstrapWitnesses :: Natural -> Natural
+        sizeOf_BootstrapWitnesses 0 = 0
+        sizeOf_BootstrapWitnesses n = 4 + 180 * n
+
+        feePerByte :: Natural
+        feePerByte = case era of
```

Comment:
Suggestion: perform the conversion earlier, which allows the type of `feePerByte` to correctly reflect what it is.

```suggestion
        feePerByte :: Coin
        feePerByte = Coin.fromNatural $ case era of
```

Reply:
Note that this is the `Ledger` coin type, and there doesn't seem to be a good existing `*`, so I'm sticking to `Natural` for now.